src/features/controls.ts

```diff
@@ -170,6 +170,17 @@ export const ControlsMixin = (ModelViewerElement:
           if (changedProperties.has('cameraOrbit')) {
             this[$updateCameraOrbit]();
           }
+          if (changedProperties.has('cameraFOV')) {
+            this[$updateCameraFov]();
+          }
+        }
+
+        [$updateCameraFov]() {
+          let fov = deserializeAngleToDeg(this.cameraFOV);
+          if (fov == null) {
+            fov = deserializeAngleToDeg(DEFAULT_CAMERA_FOV);
+          }
+          this[$controls].setFov(fov as number);
```

Comment:
Nit: since you `null`-checked this value, it's sufficient to assert it is not null to the type checker with `!`: `this[$controls].setFov(fov!)`